pkg/kubectl/cmd/get/get.go

```diff
@@ -570,7 +570,8 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	w.Flush()
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
-		fmt.Fprintln(o.ErrOut, "No resources found.")
+		resourceType := args[0]
+		fmt.Fprintf(o.ErrOut, "No %s found in namespace %s.\n", resourceType, o.Namespace)
```

Comment:
After merging https://github.com/kubernetes/kubernetes/commit/b7d2478ac9875f26250ea0803d92b58fe543e4a1 the output of kubectl contains the namespace like `No resources found in %s namespace.`
So now this PR adds a resource name into the error message.
From user viewpoint, they know what they specify as a resource.
Do we need to add it?

I guessed `$ kubectl get all` could show this error and this change could be useful to know what actual resources don't exist. However `$ kubectl get all` doesn't show such resources like the following:
```
$ kubectl get all
NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   5d6h
```
So I don't see actual merit at this time.